cmake/external/gsl.cmake

```diff
@@ -8,6 +8,7 @@ FetchContent_Declare(
     GIT_REPOSITORY https://github.com/microsoft/gsl
```

Comment:
why not move GSL to the deps file too?

Reply:
Right now the file only contains https URLs. And the last field of each row is the SHA1 checksum of the downloaded file.